### PR TITLE
Done HW_19

### PR DESCRIPTION
**Pull request by MishaBeheka**

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/accessDenied.jsp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<%--
+  Created by IntelliJ IDEA.
```
Comment (Tarabarnik) [inline src/main/webapp/WEB-INF/views/accessDenied.jsp]: remove comment please

File: src/main/webapp/WEB-INF/views/accessDenied.jsp [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  Time: 14:54
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Access denied</title>
+</head>
+<body>
+<h1>Sorry but requested is denied for you </h1>
```
Comment (Tarabarnik) [inline src/main/webapp/WEB-INF/views/accessDenied.jsp]:
```suggestion
<h1>Sorry but requested page is denied for you </h1>
```

File: src/main/webapp/WEB-INF/views/accessDenied.jsp (hunk 1 above)
Comment (MishaBeheka) [reply src/main/webapp/WEB-INF/views/accessDenied.jsp]: Done

File: src/main/webapp/WEB-INF/views/accessDenied.jsp (hunk 2 above)
Comment (MishaBeheka) [reply src/main/webapp/WEB-INF/views/accessDenied.jsp]: Done

Comment (MishaBeheka) [reply src/main/webapp/WEB-INF/views/accessDenied.jsp]: Done

**Pull request merged**